lib/parse.js

```diff
@@ -73,4 +75,58 @@ module.exports = {
     }
     return resp;
   },
+  customCSS: (file, enc, env) => {
+    // Parse the contents of a file containing CSS output. For any occurences of
+    // `url(...)` with relative paths, store the paths (for later copying) and
+    // replace them with the eventual destination paths.
+    env.customCSSFiles = env.customCSSFiles || [];
+    const src_dir = file.path.substring(0, file.path.lastIndexOf('/'));
+    let css = file.contents.toString(enc);
+    const regex = /url\(([^)]*)\)/g;
```

Comment:
@davisagli @wlonk I generally hate regular expressions, so I'm going to point out a few I've added to Herman recently (this is one), in case either of you sees red flags or has suggestions.

Reply:
That looks short, sweet, and legible to me.

Reply:
Ha! I think "legible" is pretty much out of the question for a regex, but that's my own bias... 😉